src/app/fdctl/config.c

```diff
@@ -168,6 +168,7 @@ static int parse_key_value( config_t *   config,
   ENTRY_VSTR  ( ., gossip,              entrypoints                                               );
   ENTRY_BOOL  ( ., gossip,              port_check                                                );
   ENTRY_USHORT( ., gossip,              port                                                      );
+  ENTRY_USHORT( ., gossip,              shred_version                                             );
```

Comment:
Don't think this belongs here. We already have `expected_shred_version` in this file.

Reply:
It's in consensus though